docs/source/tutorials/token_classification/rank.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd 
+import numpy as np 
+from cleanlab.rank import get_label_quality_scores as main_get_label_quality_scores 
+
+def get_label_quality_scores(labels: list, 
+                             pred_probs: list, 
+                             method: str = "min", 
+                             cleanlab_method: str = "self_confidence", 
+                             param: float = 0.04, 
+                             tokens: list = None, 
```

Comment:
Let's make `tokens` the 3rd arg of this function (move it in between `pred_probs` and `method` in the arg order)